backend/api/Areas/Property/Mapping/Search/GeoJsonMap.cs

```diff
@@ -9,6 +9,67 @@ public class GeoJsonMap : IRegister
     {
         public void Register(TypeAdapterConfig config)
         {
+            config.NewConfig<Entity.Models.PropertyModel, Model.GeoJson<Model.PropertyModel>>()
+                .Map(dest => dest.Type, src => "Feature")
+                .Map(dest => dest.Geometry.Type, src => src.Location.GeometryType)
```

Comment:
No possibility of nullreference exceptions here?

Reply:
Technically I suppose yes as objects are nullable `Location`.  But this data is from EF and the `Location` will always have a value.